### ISSUE TITLE
G5 installer no longer successfully installs driver

### ISSUE THREAD

**Issue by my-other-github-account** [tomgrice/g5-cloudrig#1]
Hello,

I was using this script successfully earlier this year and it was really helpful!

Somewhere along the way though, something seems to have stopped working with it, despite following the same install process:
<img width="842" alt="Screen Shot 2022-07-12 at 12 33 07 AM" src="https://user-images.githubusercontent.com/82987814/178434951-c5ea899b-bb68-46a9-9c54-09c79f33e9fd.png">

This is on a win 2019 server AWS G5.xlarge, 200GB SSD.

Any idea what could be going wrong or how I could fix it?

Thank you!

**Comment by Photosounder** [comment]
Same problem for me, Win 2022 server g5.xlarge.

```
Installing NVIDIA vGaming Drivers
Start-Process : This command cannot be run due to the error: The system cannot find the file specified.
At line:136 char:1
+ Start-Process "$InstallDir\NVDriver\Windows\*vgaming*.exe" -ArgumentL ...
+ ~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~
    + CategoryInfo          : InvalidOperation: (:) [Start-Process], InvalidOperationException
    + FullyQualifiedErrorId : InvalidOperationException,Microsoft.PowerShell.Commands.StartProcessCommand
```

I guess the problem is that the name of the exe changed. My workaround is manual, I get the url by adding `https://nvidia-gaming.s3.amazonaws.com/` and the one contained at https://nvidia-gaming.s3.amazonaws.com/?prefix=windows/latest (currently `windows/latest/516.59-June2022-Cloud_Gaming-Windows-Guest-Drivers.zip`) which currently gives me https://nvidia-gaming.s3.amazonaws.com/windows/latest/516.59-June2022-Cloud_Gaming-Windows-Guest-Drivers.zip, run the installer myself and then in PowerShell run this:

```
Invoke-WebRequest -Uri "https://nvidia-gaming.s3.amazonaws.com/GridSwCert-Archive/GridSwCertWindows_2021_10_2.cert" -OutFile "$Env:PUBLIC\Documents\GridSwCert.txt"
```

**Comment by tomgrice** [comment]
Apologies, guys - I had no idea people even used this script anymore. Thanks!

I'm just about to push a fix which should bring back the driver install as normal.

Any problems, let me know :)

**Comment by tomgrice** [comment]
Closed with #2
